src/main/java/pl/javastart/dictionary/LinguController.java

```diff
@@ -50,35 +50,35 @@ private void executeOption(int option) {
                 close();
```

Comment:
Taka uwaga do tego switch wyzej. Moze te opcje zamiast robic jako stale to warto zrobic jako enum? utworzyc klade enum z numerem, napisac getByCode metode zeby po odczycie pobrac odpowiedni enum i tutaj juz przekazywac enum?